casepro/cases/models.py

```diff
@@ -346,9 +371,12 @@ def reopen(self, user, note=None, update_contact=True):
     def reassign(self, user, partner, note=None, user_assignee=None):
         from casepro.profiles.models import Notification
 
+        self.last_reassigned_on = now()
+        self.last_assignee = self.assignee
+        self.last_user_assignee = self.user_assignee
```

Comment:
We should probably have a test (or modify an existing test) that checks that the `last_reassigned_on`, `last_assignee`, and `last_user_assignee` are set correctly when calling `reassign`.